### ISSUE TITLE
Persistent notification not showing on Android 13?

### ISSUE THREAD

**Issue by Mrnofish** [yanus171/Handy-News-Reader#870]
**Describe the bug**
The persistent notification does not appear on Android 13

**To Reproduce**
1. Open HNR settings
2. Click on Notifications
3. Tap on Article Reading Notification to enable it
4. Optionally restart HNR

**Expected behavior**
The persistent notification should be visible in the tray, as it was on Android 11

**Smartphone (please complete the following information):**
 - Device: guacamole
 - OS: LineageOS
 - Version 20

**Additional context**
All notifications are enabled in system settings for HNR

**Comment by yanus171** [comment]
It's due to Android 13 limitation


Mrnofish ***@***.***> 19 апреля 2023 г. 18:42:24 написал:

**Comment by Mrnofish** [comment]
Of course I appreciate newer Android versions bring about the usual slew of changes, not all of which we like, unfortunately.

The reason I've reported this, however, is that I'm running other apps that also use persistent notifications, and still have them under A13. NetGuard, Conversations, for instance, but I could name more.

Now your reply is sounding like HNR won't be able to use a persistent notification anymore, unlike those other apps.

My understanding is that the persistent notification, among other things, guaranteed that no refreshes were skipped as a result of the app being stopped by the OS.

In other words, guaranteed refreshes are going to be lost permanently?

**Comment by yanus171** [comment]
To use persistent notification like other apps, I should declare my app too 
be of special type (conservation, system etc), but it does not fall in any 
special category. If I miss with category the app could be removed from 
Google play ... But last few months I have no ability to enter Google play 
console from Russia... So may be it doesn't matter much...

Anyway, persistent notification is not concerned with guaranteed refreshes. 
It's purpose is to start app faster, But on newer Android versions it may 
have no such an effect.

**Comment by Mrnofish** [comment]
I have been experimenting with the Battery Permission and it does look like it is now necessary to set Handy to Unrestricted to avoid missing refreshes. Switching to Optimized has resulted in feeds losing posts, since most news RSS feeds only offer a limited backlog, unfortunately, making necessary to synchronize every few hours.

AntennaPod appears to be able to honor refreshes even when set to Optimized (this is for 2.7.1 as I haven't yet a chance to upgrade to 3.x).

**Comment by yanus171** [comment]
What is your HNR app version?

**Comment by Mrnofish** [comment]
The latest, 0.18.18.
